types/rest/rest.go

```diff
@@ -211,3 +219,51 @@ func PostProcessResponse(w http.ResponseWriter, cdc *codec.Codec, response inter
 	w.Header().Set("Content-Type", "application/json")
 	_, _ = w.Write(output)
 }
+
+func ParseHTTPArgs(r *http.Request) (tags []string, page, limit int, err error) {
```

Comment:
Godoc comment please, this is a public API

Reply:
Right, should be fine now, I just added a little description